tests/integration/test_install_twists.py

```diff
@@ -333,3 +334,55 @@ def test_install_local_uri_special_character(PipenvInstance, testsroot):
         c = p.pipenv("install")
         assert c.return_code == 0
         assert "six" in p.lockfile["default"]
+
+
+@pytest.mark.files
+@pytest.mark.needs_internet
```

Comment:
Does it really need internet? You are using the mocked pypi test server in this case.

Reply:
No, you're right it does not in the final iteration. Will fix that.